src/wheel/_bdist_wheel.py

```diff
@@ -243,9 +246,9 @@ def finalize_options(self) -> None:
     @property
     def wheel_dist_name(self) -> str:
         """Return distribution full name with - replaced with _"""
-        components = (
-            safer_name(self.distribution.get_name()),
-            safer_version(self.distribution.get_version()),
+        components: tuple[str, ...] = (
+            safer_name(self.distribution.get_name()),  # type: ignore[attr-defined]
+            safer_version(self.distribution.get_version()),  # type: ignore[attr-defined]
```

Comment:
```suggestion
            safer_version(
                self.distribution.get_version()  # type: ignore[attr-defined]
            ),
```

Reply:
If you don't want to use `B950` then I'll add the line breaks, but it isn't going to look pretty.

Reply:
I'll think about it but for now please do that.

Reply:
I already pushed fixes, it's just taking a long time for them to show up for some reason.
The Flake8 configuration doesn't have a local selection so it's falling back to my system settings, otherwise I wouldn't have intentionally pushed the code that violated `E501`.

Reply:
> I already pushed fixes, it's just taking a long time for them to show up for some reason.

I don't think GitHub received any fixes. They would've shown up by now.

> Flake8 doesn't have a project configuration

What about this? https://github.com/pypa/wheel/blob/main/setup.cfg#L56-L57

Reply:
It turns out I had a really old version of Flake8 and the newer versions don't use a global config anymore specifically to prevent my current issue.  I'd still consider setting the following:
```ini
[flake8]
max-line-length = 88
select = B,B9,E,F,C9
extend-ignore = E203
```